config.js

```diff
@@ -49,6 +49,7 @@ const commentsFile      = !newDeployment && loadFromFile(pe.PLUGIN_COMMENTS_FILE
 const comments          = pe.PLUGIN_COMMENTS || pe.SNOW_COMMENTS || commentsFile;
 const deploymentOutcome = /^success$/i.test(pe.PLUGIN_DEPLOYMENT_OUTCOME || pe.SNOW_STATUS || pe.DRONE_BUILD_STATUS);
 const newChange         = newDeployment || !(statusUpdate || comments);
+const failOnError       = !/^false$/i.test(pe.PLUGIN_FAIL_ON_ERROR || pe.SNOW_FAIL_ON_ERROR);
```

Comment:
I'm sure there are other tools that have a similar feature. (Though I can't think what. cURL?)

I wonder what they call it.

I wonder whether "ALLOW_FAILURE" might be clearer.